src/app/app.component.ts

```diff
@@ -1,10 +1,81 @@
-import { Component } from '@angular/core';
-
+import { Component, OnInit  } from '@angular/core';
+import { discardPeriodicTasks } from '@angular/core/testing';
+import { Injectable, Pipe, PipeTransform } from '@angular/core';
+import{SearchFilterPipe} from './filter.pipe'
 @Component({
   selector: 'app-root',
+
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  styleUrls: ['./app.component.css'],
+
 })
-export class AppComponent {
-  title = 'app';
+export class AppComponent implements OnInit{
+  tasks: Task[] = [];
+  newTask = "  ";
+  id = 1;
+  itemFilter: any = { text: '' };
+  tasksDone = [];
+   title:String;
+constructor() { 
+  this.title = " ";
+  this.tasks = []
+}
+
+ngOnInit() {
+}
+
+
+
+  pushTask = function () {
+    // jesli ten element jest pusty dodaje nowy task
```

Comment:
To co napisałeś wcale nie opisuje działania tego IFa ;)
+ W kodzie nie powinno być nic po polsku, to ogólnie przyjęta praktyka, że korzysta się z angielskiego.

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/8ea1a6acecf7953a607f0467fefd8feb509567a0